SUPPORT.md

```diff
@@ -1,19 +1,12 @@
 # Support Policy
 
-Version 1 of the Terraform Plugin SDK is considered **deprecated**, which means
+Version 1 of the Terraform Plugin SDK is considered **end-of-life**, which means
```

Comment:
Just had another thought - should we include a date here @redeux ? "As of Aug 1, 2021, Version 1 of the Terraform Plugin SDK is considered end of life."